RadixWallet/Features/DerivePublicKeys/DerivePublicKeys.swift

```diff
@@ -448,19 +448,22 @@ extension DerivePublicKeys {
 			derivationPathScheme: derivationPathScheme,
 			networkID: maybeNetworkID
 		)
+		let hardened = try index.asHardened()
```

Comment:
don't we ever derive from unhardened indices?

Reply:
No we don't, since ALL CAP26 (Babylon Account, Babylon Persona, GetID for FactorSourceID) paths MUST harden ALL components due to [SLIP10 requirements](https://github.com/satoshilabs/slips/blame/master/slip-0010.md#L96)

And for BIP44, we use a BIP44 **like** path, yours truely, incorreectly, accidentally hardened the last path component, in Olympia.

(Unfortunately it becomes a little bit more complex, since the Ledger Hardware Devices, on device, chose to correct my mistake, so even if we say HARDEN the last path component of a BIP44 path, those devices unhardened it, so if a user used a Ledger Nano S with Olympia, and then imported the same mnemonic as an Olympia Seed Phrase for a Device FactorSource that user will have problems, but we have not encountered any such user yet.)